subsys/shell/modules/Kconfig

```diff
@@ -9,6 +9,7 @@
 
 config KERNEL_SHELL
 	bool "Enable kernel shell"
+	default n
```

Comment:
the 'n' is the default if nothing else defined, so pleas remove this line.

Reply:
@nvlsianpu : I am using config FILE_SYSTEM_SHELL & not KERNEL_SHELL, so it is not related to my code changes...
"KERNEL_SHELL" is for kernel shell module & not FS shell

Reply:
Anyway your patch adds this line. Can you remove the modification?

Reply:
done